src/main/java/ch/wisv/areafiftylan/controller/TeamRestController.java

```diff
@@ -156,9 +155,8 @@ public Team getTeamById(@PathVariable Long teamId) {
             "and @currentUserServiceImpl.hasAnyTicket(#username)")
```

Comment:
Does a user that is to be invited need to have a ticket? I know this isn't implemented in this PR but I haven't seen this before. I thought the idea was that you could form a team, and then buy tickets for your team.